src/Stryker.Core/Stryker.Core/Options/StrykerOptions.cs

```diff
@@ -36,5 +34,25 @@ private string ValidateReporter(string reporter)
             }
             return reporter;
         }
+
+        private LogEventLevel ValidateLogLevel(string levelText)
+        {
+            switch (levelText?.ToLower() ?? "")
+            {
+                case "error":
+                case "":
```

Comment:
I would expect the default to still be `Warning` like it was before.

Reply:
My bad! Our documentation says error is the default, while warning was the actual default.

I think warning should be the default though. I'll ask around what others think.

Reply:
@Mobrockers and @yaelkeemink agree with me that the default should be `Warning` so we have a majority 🎉 

@Valdas3 Could you make `Warning` the default again and update the README file(s) so they also represent this?

Reply:
agreed :)

Reply:
Sure